sc/social/like/subscribers.py

```diff
@@ -148,9 +149,31 @@ def check_sharing_best_practices_on_editing(obj, event):
     state = api.content.get_state(obj)
     if state in ('published', ):
         check_sharing_best_practices(obj)
+        prefetch_facebook(obj)
 
 
 def check_sharing_best_practices_on_publishing(obj, event):
     """Event subscriber for content being published."""
     if event.status['review_state'] in ('published', ):
         check_sharing_best_practices(obj)
+        prefetch_facebook(obj)
+
+
+def prefetch_facebook(obj):
+    """Prefetching in object if enable."""
+
+    record = ISocialLikeSettings.__identifier__ + '.facebook_prefetch_enable'
+    prefetch_enable = api.portal.get_registry_record(record, default=False)
+
+    if not prefetch_enable:
+        return
+
+    url = obj.absolute_url()
+    r = requests.post('https://graph.facebook.com/?id=' + url + '&scrape=true',
+                      timeout=10,
```

Comment:
10 seconds is way too much; please use 5.

Reply:
Ok